src/components/ThemeModeSwitcher/index.jsx

```diff
@@ -0,0 +1,42 @@
+/* Temporary component for testing theme implementation and dark mode.  Used in the right side
+nav menu currently.
+
+
+@TODO convert to
+*/
+import { useEffect, useState } from 'react';
+import {
+  //   Experimental_CssVarsProvider as CssVarsProvider,
+  useColorScheme,
+} from '@mui/material/styles';
+import styles from './modeSwitcher.module.css';
+
+export const ModeSwitcher = () => {
+  const { mode, setMode } = useColorScheme();
+  const [mounted, setMounted] = useState(false);
+
+  //Need to figure out how to properly reference the theme provided by the CssVarsProvider context.
+  console.log('Called mode switcher');
+
+  useEffect(() => {
+    setMounted(true);
+  }, []);
+  if (!mounted) {
+    return null;
+  }
```

Comment:
```suggestion
  const [mounted, setMounted] = useState(false);

  //Need to figure out how to properly reference the theme provided by the CssVarsProvider context.
  console.log('Called mode switcher');

  useEffect(() => {
    setMounted(true);
  }, []);
  if (!mounted) {
    return null;
  }
```

can you check if mounted will ever "not" be true? I don't think your component will ever be not mounted and this would result in this entire section being redundant and `mounted` being a pointless state variable